parts/kubernetesmastercustomscript.sh

```diff
@@ -127,6 +127,11 @@ function ensureKubelet() {
     systemctl restart kubelet
 }
 
+function extractKubectl(){
+    systemctl enable kubectl-extract
```

Comment:
Is there an advantage to keeping it as a systemd unit rather than just executing the `docker run` and `mv` here? Does this all work out okay even when the master is rebooted (since this is `enable`d as well as started?)?